pkgs/data/fonts/rounded-mgenplus/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, p7zip }:
+
+let
+  pname = "rounded-mgenplus";
+  version = "2015-06-02";
+
+in
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
+  inherit version;
+
+  src = let v = stdenv.lib.replaceStrings ["-"] [""] version; in
+  fetchurl {
+    url = "https://osdn.jp/downloads/users/8/8598/${pname}-${v}.7z";
+    sha256 = "1k15xvzd3s5ppp151wv31wrfq2ri8v96xh7i71i974rxjxj6gspc";
+  };
+
+  nativeBuildInputs = [ p7zip ];
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  unpackPhase = ''
+    7z x $src
+  '';
+
+  installPhase = ''
+    install -m 444 -D -t $out/share/fonts/${pname} ${pname}-*.ttf
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A Japanese font based on Rounded M+ and Noto Sans Japanese";
+    homepage = http://jikasei.me/font/rounded-mgenplus/;
+    license = licenses.ofl;
+    platforms = platforms.all;
+    maintainers = with maintainers; [ ];
```

Comment:
Would you like to be a maintainer of this package? If yes, please add yourself to `maintainers/maintainer-list.nix` and to this list.